components/feature/session/src/main/java/mozilla/components/feature/session/TrackingProtectionUseCases.kt

```diff
@@ -85,7 +85,9 @@ class TrackingProtectionUseCases(
          * Removes all domains from the exception list.
          */
         operator fun invoke() {
-            engine.trackingProtectionExceptionStore.removeAll()
+            engine.trackingProtectionExceptionStore.removeAll(sessionManager.all.map {
```

Comment:
As we are applying this on all the sessions it could cause a performance hit, right now we don't have a straightforward way to avoid it, as it will required that we introduce the browser store, we are going to address this on https://github.com/mozilla-mobile/android-components/issues/6283. Would you mind adding a comment with [this link](https://github.com/mozilla-mobile/android-components/issues/6283) please ? :)